datalake/src/main/java/com/sequenceiq/datalake/service/sdx/start/SdxStartService.java

```diff
@@ -167,7 +167,7 @@ public void waitCloudbreakCluster(Long sdxId, PollingConfig pollingConfig) {
 
     private void checkFreeipaRunning(String envCrn) {
         DescribeFreeIpaResponse freeipa = freeipaService.describe(envCrn);
-        if (freeipa != null && !freeipa.getStatus().isAvailable()) {
+        if (freeipa != null && !freeipa.getAvailabilityStatus()) {
```

Comment:
Might be an obvious question but if FreeIPA is null does that not inherently mean it is not in an available state? Or is this verified someplace else.

Reply:
Good point, there are probably several other places that should have the check too.